.git_hooks/pre-commit

```diff
@@ -17,3 +17,7 @@ if [ -n "${INTERNAL_URLS}" ]; then
   echo >&2 "${INTERNAL_URLS}"
   exit 1
 fi
+
+if [ "${COMMIT_CHECK}" == "true" ]; then
```

Comment:
We are adding a new option. There are couple of scenarios for it:
- How would I know about this option?
- How would a new employee know about it?

Let's add it to docs. 

We have also other options, especially for Avito internal usage - `GRADLE_CACHE_NODE_HOST` and so on. Ideally, I'd like to see them in one place, like "Contributing/How to start" page.

Reply:
added, please take a look

Reply:
Great, thanks!